django/econsensus/publicweb/static/css/styles.css

```diff
@@ -441,6 +451,11 @@ input {
 	color: #000;
 }
 
+.summary-list td a.toggle {
```

Comment:
1) As above

2) I'm wondering if there's a more efficient way of doing this but not too bothered.